### ISSUE TITLE
Le contenu des e-mails copiés-collés dans la plateforme perdent leur mise en page lors de l'envoi

### ISSUE THREAD

**Issue by CathScott** [CCSDForge/episciences#132]
TheoretiCS :
Les rédacteurs en chef utilisent souvent la plateforme Episciences pour envoyer manuellement des courriels aux rédacteurs assignés à un article (en cliquant sur l'icône d'envoi à côté du nom du rédacteur sur la page d'administration d'un article).
Mais en faisant cela, il arrive que le message envoyé aux éditeurs perde ses sauts de ligne et mise en page.
Voir par exemple :
https://theoretics.episciences.org/administratemail/view?id=138847 (par Javier)
(Si cela peut vous aider, nous pensons que le problème peut être lié au copier-coller, puisque les messages en question sont initialement composés en collant du texte dans la boîte de message).
![image](https://user-images.githubusercontent.com/91665367/146533238-2f3b3d0c-0727-4ffd-96bf-0eb9aa14d8ec.png)

**Comment by a3nm** [comment]
Hi,

we have run into this problem again, and I now understand how to reproduce it. The actual issue is that if you try to send two messages to editors without refreshing the page, then the second message will be misformatted.

I tested on the preprod instance here: https://theoretics-preprod.episciences.org/administratepaper/view?id=1413 where we have an article assigned for testing to two accounts in my name, "Antoine Amarilli" and "Antoine Amarilli (editor)".

I first click on the email icon next to "Antoine Amarilli" in the "Editors" pane and type the message:

> line 1
> line 2

Here is what I get. Note that there is a rich text formatting widgets (buttons for bold, italic, underline, etc.)

![2022-01-05-110604_5120x1264_scrot](https://user-images.githubusercontent.com/843551/148199628-9e9801d8-78e8-450b-b8ca-8b7855228e67.png)

I click send and get a confirmation "Your email has been sent."

Now I click on the email icon next to "Antoine Amarilli (editor)" and type the same message. I get this, note that the message composition widget no longer has rich text formatting buttons.

![2022-01-05-110829_5120x1264_scrot](https://user-images.githubusercontent.com/843551/148199906-79e0bbd3-af29-49b4-89e2-45e164884208.png)

Now we see that the first message was sent out correctly:

![2022-01-05-110908_5120x1264_scrot](https://user-images.githubusercontent.com/843551/148200010-dfd1ec34-c5f2-4ffc-8da8-8c3fd42644d9.png)

But the second one was not:

![2022-01-05-110915_5120x1264_scrot](https://user-images.githubusercontent.com/843551/148200053-e392f252-373a-43f4-8aa5-b1ca4f848cae.png)

Reloading the page between the two messages is a workaround, but of course this is very confusing.

In summary, the bug is that when trying to send two editor messages without reloading the page, the second message is shown with a compose window where the rich text editing widget is broken, and the sent message then loses all line breaks.

**Comment by a3nm** [comment]
Hi, this problem has just occurred again -- and another one of our editors has received an incomprehensible message.

Do you think it would be possible to fix this issue?

Thanks in advance!

**Comment by a3nm** [comment]
Hi, asking again -- is it possible to fix this? Thanks!

**Comment by a3nm** [comment]
Thanks for addressing this!
